src/Services/Room/index.js

```diff
@@ -2,20 +2,9 @@ import Model from 'metronom';
 import MessageService from '../Message';
 import generateId from '../../Utilities/GenerateId';
 import { USER_TYPES } from '../../Constants';
+import room from '../../Model/room'
```

Comment:
```suggestion
import { Room }from '../../Models';
```

Tüm modeller `Models``klasörünün altında `index.js` içinde `import `edilip oradan toplu şekilde `export` edilebilir böylece birden fazla modeli kullanmak istediğimizde her model için yeni satır eklemez tek satırda import edebiliriz.